querydsl-core/src/main/java/com/querydsl/core/types/expr/Wildcard.java

```diff
@@ -36,12 +36,12 @@
     public static final NumberExpression<Long> count = NumberOperation.create(Long.class, Ops.AggOps.COUNT_ALL_AGG);
     
     /**
-     * Wilcard count distinct expression ((count(distinct *))
+     * Wildcard count distinct expression ((count(distinct *))
      */
     public static final NumberExpression<Long> countDistinct = NumberOperation.create(Long.class, Ops.AggOps.COUNT_DISTINCT_ALL_AGG);
 
     /**
-     * Wilcard count distinct expression ((count(distinct *))
+     * Wildcard count distinct expression ((count(distinct *))
      */
     public static final NumberExpression<Integer> countAsInt = NumberOperation.create(Integer.class, Ops.AggOps.COUNT_ALL_AGG);
```

Comment:
This is not count distinct :open_mouth: 

[see above](https://github.com/querydsl/querydsl/pull/1263/files#diff-daaa8ce4559ba899a54b3bc40fe9a8d9R34)